gnomad/utils/vcf.py

```diff
@@ -1243,20 +1250,58 @@ def make_vcf_filter_dict(
             f"{variant_qc_filter} is not a valid value for 'variant_qc_filter'. It must"
             " be 'RF' or 'AS_VQSR'"
         )
+    if (
+        not joint
+        and snp_cutoff is None
+        or indel_cutoff is None
+        or inbreeding_cutoff is None
+    ):
+        raise ValueError(
+            "snp_cutoff, indel_cutoff, and inbreeding_cutoff must be specified to generate filter descriptions."
+        )
 
-    filter_dict = {
-        "AC0": {
-            "Description": (
-                "Allele count is zero after filtering out low-confidence genotypes (GQ"
-                " < 20; DP < 10; and AB < 0.2 for het calls)"
-            )
-        },
-        "InbreedingCoeff": {
-            "Description": f"Inbreeding coefficient < {inbreeding_cutoff}"
-        },
-        "PASS": {"Description": "Passed all variant filters"},
-        variant_qc_filter: variant_qc_filter_dict[variant_qc_filter],
-    }
+    if joint:
+        filter_dict = {
+            "PASS": {
+                "Description": "Either passed all variant filters in both exomes and "
+                "genomes, or passed all variant filters in either "
+                "exomes or genomes while being absent from the other "
+                "dataset"
+            },
+            "EXOMES_FILTERED": {
+                "Description": "Failed variant filters in the exomes dataset and "
+                "either passed all variant filters in the genomes "
+                "dataset or the variant was not present in the "
+                "genomes dataset. Refer to 'exomes_filters' within "
+                "INFO for more information"
+            },
+            "GENOMES_FILTERED": {
+                "Description": "Failed variant filters in the genomes dataset and "
+                "either passed all variant filters in the exomes "
+                "dataset or the variant was not present in the exomes "
+                "dataset. Refer to 'genomes_filters' within INFO for "
+                "more information"
```

Comment:
Sorry why is this formatted into so many smaller lines instead of the regular 88 max? Did something change?

Reply:
I had 88 max for my coding style, but when I use pre-commit, it reformatted again....

Reply:
Let me just copy yours.

Reply:
I wonder if the new black is doing something strange then if its pre-commit formatting

Reply:
I think it's because of my hard-wrapping on 88 max coding style.

Reply:
Should look good now.